analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RemoveObsoleteCode.vb

```diff
@@ -0,0 +1,18 @@
+﻿Imports System
+
+Namespace Tests
+
+    <Obsolete> ' Noncompliant ^6#8 {{Do not forget to remove this deprecated code someday.}}
+    Public Class Program
+
+        <Obsolete("Message")>                   ' Noncompliant
+        Public Delegate Sub CloseDelegate(ByVal sender As Object, ByVal eventArgs As EventArgs)
```

Comment:
ByVal is obsolete in itself. Please remove.

Reply:
Yes. That's right. [converter.telerik.com](https://converter.telerik.com/) always adds these.